.github/workflows/frontend.yaml

```diff
@@ -48,3 +48,26 @@ jobs:
 
             - name: Check if the compilation is up to date
               run: git diff --no-patch --exit-code
+
+    test:
+        name: Vitest
+
+        runs-on: ubuntu-latest
+
+        steps:
+            - name: Checkout
+              uses: actions/checkout@v3
+
+            - name: Install Node
+              uses: actions/setup-node@v3
+              with:
+                  node-version: '18'
+
+            - name: Install NPM dependencies
+              uses: bahmutov/npm-install@v1
+
+            - name: Run Vitest
+              run: npx vitest run --coverage
+
+            - name: Send coverage to Codecov
+              uses: codecov/codecov-action@v3
```

Comment:
I have included coverage here, it seems to play nice with the already coverage from php files.

Reply:
Also coverage is increased but because of the Js file, I need to check coverage on php side.